libs/i18n/src/locales/en/trading.json

```diff
@@ -986,5 +986,12 @@
 
   "LIQUIDITY_TABLE_PARTY": "Party ID",
   "LIQUIDITY_TABLE_COMMITMENT": "Total commitment",
-  "LIQUIDITY_TABLE_FEES": "Fees earned (24h)"
+  "LIQUIDITY_TABLE_FEES": "Fees earned (24h)",
+
+  "// COMPETITION": "",
```

Comment:
Alphabetical please

Reply:
Partially done, I'll be moving the amm related keys in other PR